### PR TITLE
Cldc 1433 csv output symbols correctly

### PR DESCRIPTION
**Pull request by natdeanlewissoftwire**
Tested locally
Minimal risk
No docs required - variable name makes this clear.

### CODE REVIEW

File: app/controllers/case_logs_controller.rb [hunk 1]
```diff
@@ -13,6 +13,7 @@ def index
 
     all_logs = current_user.case_logs
     unpaginated_filtered_logs = filtered_case_logs(filtered_collection(all_logs, search_term))
+    byte_order_mark = "\uFEFF"
```
Comment (baarkerlounger) [inline app/controllers/case_logs_controller.rb]: Wonder if this is worth DRY-ing? I assume we'd want it anywhere we download a csv?

File: spec/requests/case_logs_controller_spec.rb [hunk 2]
```diff
@@ -755,7 +755,7 @@
 
       it "downloads a CSV file with headers" do
         csv = CSV.parse(response.body)
-        expect(csv.first.first).to eq("id")
+        expect(csv.first.first).to eq("\uFEFFid")
```
Comment (baarkerlounger) [inline spec/requests/case_logs_controller_spec.rb:758]: Might be worth still leaving in:

```ruby
        expect(csv.first.second).to eq("id")
```

**Review by baarkerlounger — APPROVED** [review]

File: spec/requests/case_logs_controller_spec.rb (hunk 2 above)
Comment (natdeanlewissoftwire) [reply spec/requests/case_logs_controller_spec.rb:758]: The second header is "status", the first header is "\uFEFF" + "id" - I don't think it's worth testing the second is it?

Comment (baarkerlounger) [reply spec/requests/case_logs_controller_spec.rb:758]: Ah gotcha, missed that

File: app/controllers/case_logs_controller.rb (hunk 1 above)
Comment (natdeanlewissoftwire) [reply app/controllers/case_logs_controller.rb]: Good point, I've put byte_order_mark into application_controller instead :)

**Pull request merged**